core/styles/components.css

```diff
@@ -214,3 +214,44 @@
   width: 100%;
   text-align: center;
 }
+
+/* toggle button */
+
+input[type='checkbox'] {
+  position: relative;
+  width: 40px;
+  height: 20px;
+  appearance: none;
+  border-radius: 2rem;
+  cursor: pointer;
+  color: red;
+  border: 2px black solid;
+  padding-right: 4px;
+}
+
+input[type='checkbox']::before {
+  content: '';
+  margin: 2.5px;
+  width: 12px;
+  height: 12px;
+  border-radius: 50%;
+  background: red;
+  position: absolute;
+  transition: 0.3s;
+}
+
+input[type='checkbox']:checked::before {
+  transform: translateX(175%);
+}
+.checkbox-container {
+  display: flex;
+  margin: 7px;
```

Comment:
replace `margin: 7px;` with `flex-direction: row;` and add `justify-content: center;`